config/locales/en.yml

```diff
@@ -1,5 +1,28 @@
 ---
 en:
+  call_for_evidence:
+    and: and
+    another_website_html: This call for evidence %{closed} held on <a href="%{url}">another website</a>
```

Comment:
Agreed that this is definitely not ideal, I'm going to take this back to Mike Whitehouse (Publishing Experience Product Owner) to get a steer on when we want to deal with this. I think a separate PR would probably be the best option as the scope of this PR is already much larger than we would like.

Reply:
Sounds good about a follow up PR, it does look like some of these have become extremely granular (perhaps the original author didn't know about the `_html` option?

Should we add something to the commit message to explain they are the way they are because of going for consistency in approach with consultation?